src/AppKit/NSWindow.cs

```diff
@@ -24,10 +24,15 @@
 using XamCore.CoreFoundation;
 using XamCore.Foundation;
 using XamCore.ObjCRuntime;
+using XamCore.CoreGraphics;
 
 namespace XamCore.AppKit {
 
 	public partial class NSWindow {
+
+		public static bool ForceReleasedWhenClosed = true;
```

Comment:
I think the naming is confusing here (when `ForceReleasedWhenClosed` is `true`, you set `ReleasedWhenClosed` to `false` - sounds like it's doing the opposite of what it's saying).